code/__DEFINES/role_preferences.dm

```diff
@@ -98,6 +99,7 @@ GLOBAL_LIST_INIT(special_roles, list(
 	ROLE_SPACE_PIRATE,
 	ROLE_FUGITIVE,
 	ROLE_FUGITIVE_HUNTER,
+	ROLE_MONKEY_HELMET,
```

Comment:
Remove it from here.
We're planning to change midround popup to be shown to everyone always (until they set it poll-ignore)